src/shims/unix/fs.rs

```diff
@@ -1659,6 +1659,60 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             }
         }
     }
+
+    #[cfg(not(windows))]
+    fn realpath(
```

Comment:
Why did you disable this on windows? Miri is supposed to present the same api's to the target program independent of the host to allow for cross-target emulation.

Reply:
It is temporary until I can implement for windows...right now it has some libc-isms I need to gate.